xclip.c

```diff
@@ -465,7 +465,10 @@ doOut(Window win)
 		else {
 		    /* no fallback available, exit with failure */
 		    char *atom_name = XGetAtomName(dpy, target);
-		    fprintf(stderr, "Error: target %s not available\n", atom_name);
+		    fprintf(stderr, "xclip::Error: Cannot performance ConvertSelection to target %s\n", atom_name);
+		    if (fverb == OVERBOSE) {
+		        fprintf(stderr, "xclip::Error: Onwer of selection: 0x%lx\n", XGetSelectionOwner(dpy, sseln));
```

Comment:
When is this useful? I think that -verbose should be used for verbose messages for the end user; not debug messages for developers.

Reply:
OK.
I have removed the check for OVERBOSE.
But I think that printing the owner id is helpful for clarifying the error for the user. For example, I can see if there is no more a owner for the selection or what program cannot fulfill the request for ConvertSelection.